leveller.js

```diff
@@ -61,16 +61,24 @@
     var diff = targetHeight - currentHeight;
     if (diff === 0) return;
 
+    var isHeightProperty = (this.options.cssProperty.indexOf('eight') > 0);
```

Comment:
Ah! It occurs to me maybe you are looking for `[hH]eight` in the string? This might be a good candidate for a regex if so.